modules/ROOT/pages/showcase-apps/service-setup.adoc

```diff
@@ -1,24 +1,28 @@
 include::{partialsdir}/attributes.adoc[]
 
+
+
 = Using the Showcase Apps
 
+
 Showcase apps demonstrate many features of {product-name}. To see these features:
 
 . Set up the appropriate services.
 . Set up the mobile app as described in xref:setting-up-the-showcase-app[].
 . Register your showcase app as described in xref:registering-a-mobile-app.adoc[Registering a Mobile App].
 . Bind your {mobile-client} to the appropriate services as described in xref:binding[]
-. Copy the `mobile-services.json` file to your IDE.
+. Copy the `mobile-services.json` file to your IDE as described in xref:downloading-the-mobile-services-configuration-file[]
 . Build and run your showcase app  as described in xref:building-and-deploying-the-showcase-apps[].
 
 [[setting-up-mobile-services-to-demonstrate-showcase-apps]]
 == Setting up Mobile Services to Demonstrate Showcase Apps
 
-
+[discrete]
```

Comment:
What does `discrete` do?

Reply:
shared answer in other channel
 [discrete] suppresses a heading so that it doesn't appear in a toc
one of things I do when editing an adoc is add :toc: temporarily to the start of a doc, then I can see if the flow of the document works while I'm editing
[discrete] stops things like == Next Steps from cluttering that view